Extensions/Today/TodayViewController.swift

```diff
@@ -158,19 +188,23 @@ class TodayViewController: UIViewController, NCWidgetProviding {
     @objc func onPressOpenClibpoard(view: UIView) {
         if let urlString = UIPasteboard.generalPasteboard().string,
             _ = NSURL(string: urlString) {
-            openContainingApp(urlString)
+            openContainingApp("firefox://?url=\(urlString)")
```

Comment:
This needs escaping. See https://github.com/mozilla/firefox-ios-open-in-client/blob/master/OpenInFirefoxClient/OpenInFirefoxControllerSwift.swift#L29